Resources/config/symfony_services.yml

```diff
@@ -22,10 +22,15 @@ services:
             - ~
             - false
 
+    auto1.api.response.transformer.name_converter.decorator.hyphen_to_underscore_converter:
+        class: Auto1\ServiceAPIComponentsBundle\Service\NameConverter\HyphenToUnderscoreNameConverterDecorator
+        arguments:
+            - '@auto1.api.response.transformer.name_converter.camel_case_to_snake_case'
+
     auto1.api.response.transformer.name_converter.decorator.skip_convert_on_normalize:
         class: Auto1\ServiceAPIComponentsBundle\Service\NameConverter\SkipConvertOnNormalizeNameConverterDecorator
         arguments:
-            - '@auto1.api.response.transformer.name_converter.camel_case_to_snake_case'
+            - '@auto1.api.response.transformer.name_converter.decorator.hyphen_to_underscore_converter'
```

Comment:
it does look like breaking change, impacting current behavior of applications

Reply:
@Dropaq If somebody has overridden these private services, it could impact. This solution looks less dangerous than making this bundle configurable.
I can try to add some functional test in this bundle to ensure that everything works as expected.
I'm open for suggestion.

Reply:
@aviator-ua I think what @Dropaq meant - is behaviour of `auto1.api.response.transformer.name_converter.decorator.skip_convert_on_normalize` service changes a bit now. And that indeed can cause issues when updating to a new version of this thing.